components/script/dom/bindings/trace.rs

```diff
@@ -430,6 +507,69 @@ impl<T> JSTraceable for IpcReceiver<T> where T: Deserialize + Serialize {
     }
 }
 
+impl JSTraceable for Matrix2D<f32> {
+    #[inline]
+    fn trace(&self, _trc: *mut JSTracer) {
+        // Do nothing
+    }
+}
+
+impl JSTraceable for Matrix4D<f64> {
```

Comment:
Well, then I'd have to trace all the elements of the matrix. I can do that if you prefer.